src/components/Quiz/CurrentQuestion/CurrentQuiz.tsx

```diff
@@ -0,0 +1,163 @@
+import { useState } from "react";
+import {Link} from 'react-router-dom';
+import { useQuiz } from "../../../QuizContext/QuizContext";
+import { Quiz } from "../../../QuizDB/QuizDB";
+import './CurrentQuiz.css'
+
+type Prop = {
+  currentQuiz: Quiz;
+};
+
+export const CurrentQuestion = ({ currentQuiz }: Prop) => {
+  const { quizState, quizDispatch } = useQuiz();
+  const [disableButtons, setDisableButtons] = useState<boolean>(false);
+  const [optionId, setOptionId] = useState<string>("");
+  const nextQuestion = () => {
+    if (currentQuiz) {
+      quizState.currentQuestionNumber === currentQuiz.questions.length - 1
+        ? quizDispatch({ type: "INCREMENT_QUESTION_NUMBER", payload: -1 })
```

Comment:
dispatch name can be more generic for example update current question, as it is both incrementing and decrementing